src/hotspot/os/aix/osThread_aix.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 1999, 2023, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2012, 2015 SAP SE. All rights reserved.
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2024 SAP SE. All rights reserved.
```

Comment:
@offamitkumar  do you work for SAP? If not then please do not modify their copyright notice. Thanks for updating the Oracle one's as I requested.